Update client-errors.mdx - Update Happy Eyeballs failure error message name and add a condition (device revocation) (#18245)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/connections/connect-devices/warp/troubleshooting/client-errors.mdx

A router, firewall, antivirus software, or other third-party security product is blocking UDP on the WARP ports.

### Resolution
 
1. Configure the third-party security product to allow the [WARP ingress IPs and ports](/cloudflare-one/connections/connect-devices/warp/deployment/firewall/#warp-ingress-ip).
2. Ensure that your Internet router is working properly and try rebooting the router.

> 3. Check that the device is not revoked in the [devices page of the dashboard](https://one.dash.cloudflare.com?to=/:account/team/devices). 

Reviewer: ```suggestion
3. Check that the device is not revoked by going to **My team** > **Devices**. 
```
